formats.sh

```diff
@@ -23,11 +25,11 @@ if [ ! ${#missing_dependencies[@]} -eq 0 ]; then
   pip install "${missing_dependencies[@]}"
 fi
 
-update=0
-while getopts "u" OPT
+update=1
+while getopts "n" OPT
```

Comment:
Silently added a dry-run option. I'm not sure it is needed. If someone think it should be removed, I'll do.